### PR TITLE
Fixup proxy docs for DOCS-1710

### PR DESCRIPTION
**Pull request by vishnu2kmohan**
## Description
Fixup no_proxy directive and make http_proxy and https_proxy examples better as described in https://jira.mesosphere.com/browse/DOCS-1710

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.7/administration/installing/custom/configure-proxy.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ By default the DC/OS [Universe](https://github.com/mesosphere/universe) reposito
...
     ```
-    http_proxy=http://user:pass@host:port
-    https_proxy=http://user:pass@host:port
-    no_proxy="*.mesos,127.0.0.1,localhost"
+    http_proxy=http://<user>:<pass>@<proxy_host>:<http_proxy_port>
+    https_proxy=https://<user>:<pass>@<proxy_host>:<https_proxy_port>
+    no_proxy=".mesos,.thisdcos.directory,.dcos.directory,.zk,127.0.0.1,localhost"
     ```
     
     If you are not sure about the values for `http_proxy` and `https_proxy` variables for your environment, Please contact your system administrator.
```
Comment (sascala) [inline 1.7/administration/installing/custom/configure-proxy.md]: ^ remove "Please"

Comment (vishnu2kmohan) [inline 1.7/administration/installing/custom/configure-proxy.md]: Done!

File: 1.8/administration/installing/custom/advanced.md [hunk 2]
```diff
@@ -82,10 +82,11 @@ The DC/OS installation creates these folders:
     - 8.8.4.4
     - 8.8.8.8
```
Comment (joel-hamill) [inline 1.8/administration/installing/custom/advanced.md]: @vishnu2kmohan please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.

File: 1.8/administration/installing/custom/configuration-parameters.md [hunk 3]
```diff
@@ -157,9 +157,9 @@ This parameter specifies whether to enable the DC/OS proxy.
 *  `use_proxy: 'false'` Do not configure DC/OS [components](/docs/1.8/overview/components/) to use a custom proxy. This is the default value.
 *  `use_proxy: 'true'` Configure DC/OS [components](/docs/1.8/overview/components/) to use a custom proxy. If you specify `use_proxy: 'true'`, you can also specify these parameters:
     
-    *  `http_proxy: <your_http_proxy>` This parameter specifies the HTTP proxy.
-    *  `https_proxy: <your_https_proxy>` This parameter specifies the HTTPS proxy.
-    *  `no_proxy: - <ip-address>` This parameter specifies YAML nested list (-) of addresses to exclude from the proxy.
+    *  `http_proxy: http://<user>:<pass>@<proxy_host>:<http_proxy_port>` This parameter specifies the HTTP proxy.
```
Comment (joel-hamill) [inline 1.8/administration/installing/custom/configuration-parameters.md]: @vishnu2kmohan please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.

File: 1.9/administration/installing/custom/advanced.md [hunk 4]
```diff
@@ -82,10 +82,11 @@ The DC/OS installation creates these folders:
     - 8.8.4.4
     - 8.8.8.8
     use_proxy: 'true'
-    http_proxy: http://<your_http_proxy>/
-    https_proxy: https://<your_https_proxy>/
+    http_proxy: http://<proxy_host>:<http_proxy_port>
```
Comment (joel-hamill) [inline 1.9/administration/installing/custom/advanced.md]: @vishnu2kmohan please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.

File: 1.9/administration/installing/custom/configuration-parameters.md [hunk 5]
```diff
@@ -200,9 +200,9 @@ This parameter specifies whether to enable the DC/OS proxy.
 *  `use_proxy: 'false'` Do not configure DC/OS [components](/docs/1.9/overview/architecture/components/) to use a custom proxy. This is the default value.
 *  `use_proxy: 'true'` Configure DC/OS [components](/docs/1.9/overview/architecture/components/) to use a custom proxy. If you specify `use_proxy: 'true'`, you can also specify these parameters:
     **Important:** The specified proxies must be resolvable from the provided list of [resolvers](#resolvers).
-    *  `http_proxy: <your_http_proxy>` This parameter specifies the HTTP proxy.
-    *  `https_proxy: <your_https_proxy>` This parameter specifies the HTTPS proxy.
-    *  `no_proxy: - <ip-address>` This parameter specifies YAML nested list (-) of addresses to exclude from the proxy.
+    *  `http_proxy: http://<user>:<pass>@<proxy_host>:<http_proxy_port>` This parameter specifies the HTTP proxy.
```
Comment (joel-hamill) [inline 1.9/administration/installing/custom/configuration-parameters.md]: @vishnu2kmohan please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.

File: 1.8/administration/installing/custom/advanced.md (hunk 2 above)
Comment (joel-hamill) [inline 1.8/administration/installing/custom/advanced.md]: @vishnu2kmohan https://github.com/mesosphere/dcos-docs-enterprise/pull/1096/files

**Pull request merged**